src/compiler/moduleNameResolver.ts

```diff
@@ -1153,4 +1158,68 @@ namespace ts {
     function toSearchResult<T>(value: T | undefined): SearchResult<T> {
         return value !== undefined ? { value } : undefined;
     }
+
+    /* @internal */
+    export const enum PackageNameValidationResult {

```

Comment:
These types shouldnt be in compiler .. They should be in Language service since they arent needed by compiler.